packages/react/src/FilteredActionList/useAnnouncements.tsx

```diff
@@ -36,17 +38,26 @@ const getItemWithActiveDescendant = (
 
   return {index, text, selected}
 }
+//TODO remove this when we remove usingRemoveActiveDescendant
 
 export const useAnnouncements = (
   items: FilteredActionListProps['items'],
   listContainerRef: React.RefObject<HTMLUListElement>,
   inputRef: React.RefObject<HTMLInputElement>,
   enabled: boolean = true,
   loading: boolean = false,
-  message?: {title: string; description: string},
+  message?: {title?: string; description?: string},
 ) => {
+  const usingRemoveActiveDescendant = useFeatureFlag('primer_react_select_panel_remove_active_descendant')
   const liveRegion = document.querySelector('live-region')
 
+  // Notify user of the number of items available
+  const selectedItems = items.filter(item => item.selected).length
+  const announcementText =
+    items.length > 0
+      ? `${items.length} item${items.length > 1 ? 's' : ''} available, ${selectedItems} selected.`
+      : `${message?.title}. ${message?.description}`
```

Comment:
I think this needs a check for loading state to avoid announing no items unnecessarily